Client/src/environments/environment.ts

```diff
@@ -4,6 +4,9 @@
 
 export const environment = {
   artistsApiUrl: 'https://localhost:7093/api/artists/',
+  topTracksApiPostfix: 'top-tracks',
```

Comment:
Let's move apiPostfixes to the corresponding services and leave only api url here.